src/main/java/mate/academy/Main.java

```diff
@@ -14,7 +16,7 @@
 public class Main {
     private static final Injector injector = Injector.getInstance("mate.academy");
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws RegistrationException, AuthenticationException {
```

Comment:
throws exception in main() method is a bad practice
let's use catch block for it